docs/070-project-management/pm-unanet-tasks.md

```diff
@@ -0,0 +1,241 @@
+
+   - You will be directed to a similar screen where you can "enable budget field for updates" and enter the hours you wish to allocate for that person for each month.
+   - You can also enter the labor category here.
+
+3. For the same number of hours for each person for a particular month - Double click on the month at the top of the page
+
+   - You will be directed to a similar screen where you can "enable budget field for updates" and enter the hours you wish to allocate for each person for the selected month.
+   - You can also enter the labor category here if it is the same for each person
+
+## Adjust projections for holidays or other time-off
+
+In general, resource allocations can be expected using the following rations:
+
+1. If a FT resource devoted to the project, then 80%
+2. If an AD resource devoted to the project, then 65%
+3. If a PT resource devoted to the project, then 40%
+
+From that place, then deduct - monthly - for any of the following:
+
+1. holidays
+2. OOO
+3. Any other events, e.g., summits, etc.
+
+Pro Tip: For finding OOO dates on the calendar, use the CivicActions shared calendar and try searching "name OOO". This will allow you to isolate the forthcoming dates for the given person.
+
+## Reviewing and approving expenses
+
+1. Find your approvals
+   1. Click on the Unanet logo
+   1. Scroll down to Approvals
+   1. Look for Expense Reports and/or Requests and click on Primary Approvals to view
+   1. ![Screenshot of Approving Expense 1](../images/Approving-Expenses-1.png)
+   1. NOTE: All expense reimbursements except for ProDev < $50 should be submitted as a REQUEST first.
+   1. To check this quickly:
+      1. Click on the magnifying glass next to the expense report
+      1. Scroll down to Approval History
+      1. A report that has previously been a request will show "PreApproving" and "PreApproved" statuses in the history. If it has never been a request, you will only see "Approving" and/or "Completed". (See below for explanation of statuses and an example of an expense request and report approval history).
+   1. To check this by viewing the person expense approval history:
+      1. People/List/Type last name in Last Name field and click blue > then List
+      1. Click on the dollar icon
+      1. Click on "Completed Expense Requests" to view
+1. Review the REQUEST
+   1. Click on the magnifying glass to view all details
+   1. Check the project type and expense type are correct
+   1. Approve or Disapprove as you see fit - you do NOT need to check ProDev usage - We do that as part of the project approval process before the request reaches you.
+1. Review the REPORT
+   1. Make sure there are no duplicates
+   1. Click on the magnifying glass to view all details
+   1. Check the amount, project type and expense type are correct
+   1. Click on attachments to open the receipt and double check the details
+   1. Checklist
+      1. Receipt should be attached
```

Comment:
@farooqzakhilwal looks like unfortunately lists 3-levels or deeper don't render well, see https://handbook.civicactions.com/en/farooqzakhilwal-patch-1/070-project-management/pm-unanet-tasks/

They all appear as:

> Checklist 1. Receipt should be attached 1. Check project/expense type 1. Check the receipt date (should match date entered in Unanet) 1. Check the receipt amount (should match amount entered in Unanet) 1. Check the credit card number does not end in xxxx. If it does, then the receipt needs to be associated with our company credit card and not reimbursed. If it is the company credit card, click on "disapprove" and ask the requestor to resubmit with "x Visa" for Payment Method (on the Details page) 1. If there are any problems with the report, click on "disapprove" and let the requestor know why you have disapproved it 1. If all looks good, click on "Approve"

An alternate solution is using sub-headings and then having lists under those.

Reply:
Sorry the handbook rendering can be a pain.